JitsiMeetJS.js

```diff
@@ -42,6 +43,9 @@ var LibJitsiMeet = {
 
     version: '{#COMMIT_HASH#}',
 
+    // expose JitsiTrackError this way to give library consumers to do checks
+    // like if (error instanceof JitsiMeetJS.JitsiTrackError) { }
+    JitsiTrackError: JitsiTrackError,
```

Comment:
JitsiConnection is corresponding more to JitsiTrack as a type of object we expose, not so much to JitsiTrackError. I've had a closer look to the code and I suggest that we expose JitsiTrackError and expose the possible error types from JitsiTrackError.